Makefile

```diff
@@ -14,22 +16,22 @@ LUA_INCLUDE_DIR ?= $(PREFIX)/include
 LUA_LIB_DIR ?=     $(PREFIX)/lib/lua/$(LUA_VERSION)
 INSTALL ?= install
 
+CARGO := $(HOME)/.cargo/bin/cargo
+TARGET ?= x86_64-unknown-linux-gnu
+
 .PHONY: all test install build clean
 
 all: ;
 
 build: target/release/libatc_router.so target/release/libatc_router.a
```

Comment:
i guess this would need to use `$(TARGET)` instead of `release`.   perhaps something like `$(addprefix target/$(TARGET)/, libatc_router.so libatc_router.a)`